src/components/pages/order/OrderPage.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { useNavigate, useLocation } from "react-router-dom"
+
+function OrderPage() {
+  
+  const Navigate = useNavigate();
+  const { state } = useLocation();
```

Comment:
```suggestion
  const { name } = useLocation();
```

choisis tout ce que tu veux comme nom, mais pas "state" qui est trop générique et vague. On veut être explicite ici, on veut savoir en un coup d'oeil ce qu'on obtient de l'URL.